diff --git a/node_modules/react-native-vision-camera-mlkit/plugin/index.js b/node_modules/react-native-vision-camera-mlkit/plugin/index.js
index 2e80c96..83aeafb 100644
--- a/node_modules/react-native-vision-camera-mlkit/plugin/index.js
+++ b/node_modules/react-native-vision-camera-mlkit/plugin/index.js
@@ -1,4 +1,4 @@
-const { createRunOncePlugin } = require('@expo/config-plugins');
+const { createRunOncePlugin } = require('expo/config-plugins');
 const { name, version } = require('../package.json');
 const { parseMLKitConfig } = require('./mlkitConfig');
 const { withVisionCameraMLKitAndroid } = require('./withAndroid');
diff --git a/node_modules/react-native-vision-camera-mlkit/plugin/withAndroid.js b/node_modules/react-native-vision-camera-mlkit/plugin/withAndroid.js
index bb2ccf5..5e6abca 100644
--- a/node_modules/react-native-vision-camera-mlkit/plugin/withAndroid.js
+++ b/node_modules/react-native-vision-camera-mlkit/plugin/withAndroid.js
@@ -1,4 +1,4 @@
-const { withProjectBuildGradle } = require('@expo/config-plugins');
+const { withProjectBuildGradle } = require('expo/config-plugins');
 
 const MARKER =
   '// react-native-vision-camera-mlkit selective ML Kit dependencies';
diff --git a/node_modules/react-native-vision-camera-mlkit/plugin/withIOS.js b/node_modules/react-native-vision-camera-mlkit/plugin/withIOS.js
index c5ef374..78fff83 100644
--- a/node_modules/react-native-vision-camera-mlkit/plugin/withIOS.js
+++ b/node_modules/react-native-vision-camera-mlkit/plugin/withIOS.js
@@ -1,4 +1,4 @@
-const { withPodfile } = require('@expo/config-plugins');
+const { withPodfile } = require('expo/config-plugins');
 const { ANDROID_ONLY_KEYS } = require('./mlkitConfig');
 
 const MARKER =
